<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TesterLightDLLClass</class>
 <widget class="QMainWindow" name="TesterLightDLLClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TesterLightDLL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonLoadDLL">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt; Load DLL</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetLightDLL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>441</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonOnOff">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetting">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveToDefault">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to default</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveAs">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadFromDefault">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>260</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load from default</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadAs">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLightParamStr">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Initial Parameter</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
